test.js

```diff
@@ -0,0 +1,24 @@
+import test from 'ava'
+import isPresent from 'is-present'
+import colorable from './'
+
+test('it returns an array of color combinations', t => {
+  t.plan(4)
+
+  const colors = {
+    red: 'tomato',
+    blue: 'blue',
+    gray1: '#333',
+    gray2: '#444'
+  }
+
+  const colorCombos = colorable(colors)
+  const gray1Combos = colorCombos[2].combinations
+  const redContrast = gray1Combos[0].contrast
+  const gray2Contrast = gray1Combos[2].contrast
+
+  t.true(isPresent(colorCombos))
```

Comment:
It doesn't unfortunately. I can modify the code to not include `is-present` and instead ensure there's a `length`. I've come to prefer the readability of the `isPresent` function call myself. But I'm happy to rework to not include that dev dependency.